app/services/PaymentHistoryService.scala

```diff
@@ -85,6 +70,16 @@ class PaymentHistoryService @Inject()(repaymentHistoryConnector: RepaymentHistor
       }
     else Future(Right(Nil))
   }
+
+  def combineTwoResponses(response1: Either[PaymentHistoryError.type, Seq[Payment]],
```

Comment:
we can move thins to response level, ie as an function in companion object, but should be fine for now to keep it here